repo/packages/M/minio/0/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "http://gsvlabs.com/wp-content/uploads/2015/08/Minio_Logo_Final_With-White-Box1.png",
+    "icon-medium": "http://gsvlabs.com/wp-content/uploads/2015/08/Minio_Logo_Final_With-White-Box1.png",
+    "icon-large": "http://gsvlabs.com/wp-content/uploads/2015/08/Minio_Logo_Final_With-White-Box1.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "minio-docker-RELEASE": "minio/minio:latest"
```

Comment:
@ryadav88 : We have decided to have docker images with tags corresponding to our releases, Will have it integrated in couple of days.